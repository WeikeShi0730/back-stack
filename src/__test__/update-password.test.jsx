import { render } from "@testing-library/react";
import UpdatePassword from "../components/update-password/update-password.component";

describe("<UpdatePassword />", () => {
  test("<UpdatePassword />", () => {
    const { baseElement } = render(<UpdatePassword />);
    expect(baseElement).toMatchSnapshot();
  });
});
