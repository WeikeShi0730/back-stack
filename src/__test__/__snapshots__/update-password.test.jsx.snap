// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<UpdatePassword /> <UpdatePassword /> 1`] = `
<body>
  <div>
    <div
      class="max-w-xs sm:max-w-sm md:max-w-md lg:max-w-lg m-auto mt-10 mb-4 bg-white rounded-lg border border-primaryBorder shadow-default py-8 px-10 text-sm md:text-base lg:text-lg"
    >
      <div
        class="flex flex-col items-center"
      >
        <div
          class="text-center m-5"
        >
          Update my password?
        </div>
        <button
          class="text-xs md:text-sm bg-gray-800 py-2 px-4 text-white rounded border focus:outline-none font-light"
        >
          Send a password update email
        </button>
      </div>
    </div>
  </div>
</body>
`;
